packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -139,7 +139,7 @@ export const render = async (req, res, next) => {
         locals: res.locals
     })
 
-    let routes = getRoutes(res.locals)
+    let routes = await getAllRoutes(res.locals)
```

Comment:
Jinsu is going to require the url I believe. Is that available in the locals?

Reply:
@bendvc yup, it's available in `locals.originalUrl`.